cross/android/.gitignore

```diff
@@ -0,0 +1,8 @@
+deb/
+tmp/
+toolchain/
+lldb/
+android-ndk-*/
+libunwind/
+*.zip
+
```

Comment:
The guidance for coreclr is to have just a single .gitignore in the root. 
I think that we should add just the cross/android/* in there, like we do for the cross/rootfs/*